git-poller/src/main/bash/git-poller.sh

```diff
@@ -64,13 +64,13 @@ function build_clone_url() {
     ENCODED_USERNAME=$(url_encode "$GIT_USER")
     ENCODED_PASSWORD=$(url_encode "$GIT_PASS")
     SCHEME=$(echo "$REPO" | sed 's|:.*||')
-    CTX_PATH=$(echo "$REPO" | sed "s|$SCHEME://$TARGET_HOST/||")
-    CLONE_URL="$SCHEME://$ENCODED_USERNAME:$ENCODED_PASSWORD@$TARGET_HOST/$CTX_PATH"
+    CTX_PATH=$(echo "$REPO" | sed "s|$SCHEME://$TARGET_HOST||")
+    CLONE_URL="$SCHEME://$ENCODED_USERNAME:$ENCODED_PASSWORD@$TARGET_HOST$CTX_PATH"
```

Comment:
@german-muzquiz idk how useful this tip is, but for path issues like this bash has this feature to remove leading or trailing slahes:

```
mypath=/something/something/

echo ${mypath##/}                                                                                                                                                                                    
# (removes leading) prints:
#     something/something/

echo ${mypath%%/} 
# (removes tailing) prints:
#     /something/something

# It doesnt affect paths without trailing or leading
myotherpath=something/something
echo ${myotherpath##/} 
echo ${myotherpath%%/} 
# both print the same:
#   something/something
```

Reply:
From this input: `http://gitea:3000/org/repo.git`, I'm trying to get `:3000/org/repo.git` for later adding it to the clone url. The bug is that sed was looking for `http://gitea/`, didn't find it and so `CTX_PATH` was the entire url.

Nice tip on bash character deletion instead of using sed!